plugins/mcreator-localization/lang/texts.properties

```diff
@@ -3059,3 +3059,5 @@ blockly.errors.duplicate_dependencies_provided.statement=Statement input {0} pro
 blockly.errors.api_required="{0}" API required by block {1} is either disabled in workspace settings or not supported by the current generator.
 blockly.errors.call_procedure_missing_deps=The procedure you are calling ({0}) requires the following dependencies that are not provided: {1}
 blockly.block.cancel_event=Cancel event
+blockly.block.entity_get_ticks_frozen=Get number ticks %1 has been frozen for
+blockly.block.entity_set_ticks_frozen=Set number of ticks %1 has been frozen for to %2
```

Comment:
Move set block after line 976 and get block after line 885.
Also get block message is missing the preposition between _number_ and _ticks_.